src/js/player.js

```diff
@@ -1286,6 +1286,11 @@ class Player extends Component {
 
     this.removeClass('vjs-ended');
     this.removeClass('vjs-seeking');
+    const loadingSpinner = this.getChild('loadingSpinner');
```

Comment:
I think it would be better to make a `showSpinner` and `hideSpinner` function and call that it each of these places.

Reply:
components are already supposed to have show/hide methods, so, I think this is fine

Reply:
@BrandonOCasey I'm trying not to make the player class feature envious, which is a code smell (https://refactoring.guru/smells/feature-envy), thus the override of existing show and hide methods is appropriate